### ISSUE TITLE
get back all old sessions from session manager to TabSessionManager after Quantum migration

### ISSUE THREAD

**Issue by Abouriou** [sienori/Tab-Session-Manager#36]
will I have to reinstall old Firefox to get back my tabs ?

**Comment by sienori** [comment]
To migrate old sessions, you must do it manually in firefox 56 or earlier.
It is only possible to activate SessionManager and TabSessionManager at the same time and restore and save sessions manually.

**Comment by Abouriou** [comment]
Thank you Sienori-San
So my intuition was not too bad

I indeed loaded version 56.0.2, the effect of which was to get back automatically my former "Session Manager" _plus_ the Tab Session Manager that I had loaded through Quantum right before. Guess what, then I had both in my add-ons and had nothing else to do manually than close firefox and upgrade back to Quantum (57). I immediately got back my 67 open tabs (I know it is bad launching that much in the same session). I just advise to counter-check Tab Session Manager's options and settings, once you have reinstalled the old Firefox.

To put it in a nutshell, thank you once again.
